web/modeshape-web-cmis/pom.xml

```diff
@@ -80,7 +80,6 @@
         <dependency>
             <groupId>org.modeshape</groupId>
             <artifactId>modeshape-jcr</artifactId>
-            <scope>test</scope>
```

Comment:
@okulikov: the code should be using `org.modeshape.jcr.api.Binary` and then this change would not be needed - i.e. it should still be possible to compile against `modeshape-jcr-api` only